spec/gemstash/preload_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "spec_helper"
+
+describe Gemstash::Preload do
+  let(:stubs) { Faraday::Adapter::Test::Stubs.new }
+  let(:http_client) { Gemstash::HTTPClient.new(Faraday.new {|builder| builder.adapter(:test, stubs) }) }
+  let(:latest_specs) do
+    to_marshaled_gzipped_bytes([["latest_gem", "1.0.0", ""]])
+  end
+  let(:full_specs) do
+    to_marshaled_gzipped_bytes([["latest_gem", "1.0.0", ""], ["other", "0.1.0", ""]])
```

Comment:
The platform seems to be "ruby" rather than an empty string when the specs are fetched from rubygems.org.

Also, I just pulled latest_specs and inspected the result... it appears that it is all gems indicating the latest version, not anything to do with when gems are pushed. Thus, latest would have both latest_gem and other, 1 version each.... full_specs on the other hand would have both, but all (non prerelease) versions of the gems.